tests/codegen/align-struct.rs

```diff
@@ -1,5 +1,6 @@
 //@ compile-flags: -C no-prepopulate-passes -Z mir-opt-level=0
-//
+// 32bit MSVC does not align things properly so we suppress high alignment annotations (#112480)
+//@ ignore-msvc
```

Comment:
Is there a way to only ignore 32bit msvc and still keep this test for 64bit msvc?

Reply:
We have full target-spec matching in compiletest now? Nice! When did that happen?

Reply:
I believe some targets are special-cased to work.

Reply:
That's rather messy :/ https://rustc-dev-guide.rust-lang.org/tests/directives.html says I can use a full target triple. I'll file an issue.